src/databricks/setup.py

```diff
@@ -16,7 +16,7 @@
 
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
-VERSION = '0.10.2'
+VERSION = '0.11.0'

```

Comment:
please update version as cli extension versioning bot suggested.

for more info, please refer to this doc: https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md